ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/dialog.component.html

```diff
@@ -2,13 +2,14 @@
 <div class="fudis-dialog fudis-dialog__size__{{ size }}" [attr.id]="_id">
 	<fudis-button
 		class="fudis-dialog__close"
+		[class.fudis-dialog__close__absolute]="closeButtonPositionAbsolute"
 		[attr.id]="_id + '-close'"
-		[label]="closeButtonLabel"
+		[label]="_closeLabel"
 		[labelHidden]="true"
-		fudisDialogClose
 		[size]="'icon-only'"
 		[icon]="'ring-close'"
-		[variant]="'tertiary'">
+		[variant]="'tertiary'"
+		fudisDialogClose>
```

Comment:
No need to change this, as this is opinionated, but at least I have tried to list all directives (fudisDialogClose) as first attributes of the element. Just trying to keep these even remotely consistent! :D